attestation-service/docs/grpc-as.md

```diff
@@ -2,15 +2,62 @@
 
 `grpc-as` is an Attestation Service application based on gRPC protocol.
 
-## API
+Now the following types of evidence are supported:
+- `azsnpvtpm`: Azure SNP vTPM
+- `sev`: AMD SEV (Not implemented)
+- `sgx`: Intel SGX
+- `snp`: AMD SNP
+- `tdx`: Intel TDX
+- `cca`: Arm CCA
+- `sample`: A fake platform. Only for test and sample
+- `csv`: Hygon CSV
+- `aztdxvtpm`: Azure TDX vTPM (Not implemented)
 
-gRPC Attestation Service provides gRPC endpoints which is defined in [protobuf](../protos/attestation.proto).
+## Quick Start
 
-## Usage
+Users can use a [community version of gRPC CoCoAS image](https://github.com/confidential-containers/trustee/pkgs/container/staged-images%2Fcoco-as-grpc) to verify attestation reports.
 
-Here are the steps of building and running gRPC Attestation Service:
+```shell
+# run gRPC CoCoAS server locally
+docker run -d \
+  -v /etc/sgx_default_qcnl.conf:/etc/sgx_default_qcnl.conf \ # this qcnl config is used when verifying SGX/TDX quotes
+  -p 50004:50004 \
+  ghcr.io/confidential-containers/staged-images/coco-as-grpc:latest
+```
+
+Then an attestation request can be used to request the server. We provide an [example request of SGX quote](../tests/coco-as/grpc-request.json).
+```shell
+# Use the following cmdline to install grpcurl
+# go install github.com/fullstorydev/grpcurl/cmd/grpcurl@latest
 
-### Build
+cd <path-to-attestation-service>
+
+REQ=$(cat tests/coco-as/grpc-request.json)
```

Comment:
How do I generate this request, for other TEEs?

Reply:
You'd better refer to `API` section to see the proto definition, and "make-up" a request due to the `message AttestationRequest`.

So how do you think I should mention this in the doc?

Reply:
I am myself, trying to get my head around. I think if it safe to just say that you can generate such a request by using something from the AA ?

Reply:
Nice idea. Fortunately we have a [tool](https://github.com/confidential-containers/guest-components/tree/main/attestation-agent/attester#evidence-getter-tool) prepared before.